torchao/quantization/GPTQ/README.md

```diff
@@ -0,0 +1,74 @@
+# GPTQ
+
+GPTQ is a quantization technique that improves accuracy for various forms of quantization as introduced in the paper: https://arxiv.org/abs/2210.17323
+
+In general GPTQ requires a model, a quantization technique and calibration data. GPTQ then optimizes the quantization parameters and quantized weights so they are more accurate accross the calibration data.
+
+## API
+
+The api for this technique is as follows:
+
+
+```python
+from torchao.quantization import MultiTensorInputRecorder, Int4WeightOnlyGPTQQuantizer
+
+model = get_model() # user provided function
+
+# first gather inputs
+input_recorder = MultiTensorInputRecorder()
+for i in range(calibration_limit):
+    args = get_next_input() # user provided function
```

Comment:
optional nit: might be useful to have some real data example (e.g. https://github.com/pytorch/ao/blob/c4250a4bfa495a260ea009e23a9b8202bb93b52d/torchao/prototype/awq/example.py#L21), but I'm also planning to take a look at this soon